vllm/model_executor/models/gpt_j.py

```diff
@@ -274,11 +274,16 @@ def load_weights(self,
             for (param_name, weight_name, shard_id) in stacked_params_mapping:
                 if weight_name not in name:
                     continue
-                param = params_dict[name.replace(weight_name, param_name)]
+                name = name.replace(weight_name, param_name)
+                if name not in params_dict:
+                    continue
```

Comment:
Let's not use a wildcard `if` here since this can lead to silent errors. Do you know which exact parameter is causing this error?

Reply:
AutoGPTQ sets bias to true for all linear layers, leading to extra all-zero bias in model weights. I'll add more condition to it.